_pages/topics/git_recovery.md

```diff
@@ -34,5 +37,29 @@ Ez egy elég általános eszköz, ami veszélyes, de nagyon hasznos ha tudod mit
 ### rebase vagy merge?
 A `merge` egy speciális commitot készít, míg a `rebase` egy commit listát szerkeszt, bizonyos értelemben átírva a "történelmet".  Merge helyett is használható, nem csak hibajavításra, mert fast-forward merge-et tesz lehetővé.
 
-### fast-forward???
-TODO describe fast forward, probably move it to basics section
+### fast-forward?
+Ha mergelés folyamán ha egy olyan commitot akarsz egy másik committal mergelni ami elérhető az első commit history-át követve, akkor a Git azzal egyszerűsíti le a dolgokat hogy a második commit head-jére állítja az első pointerét, mert nincsenek divergens változtatások amit mergelni kellene, ez a “fast-forward”.
```

Comment:
Ezt lehetne egyszerűbben is elmagyarázni, jelenleg kicsit ilyen "erőltetett dokemtáció fordítás" érzést kelt.

Meg a heading szintek itt is hibásak.